src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
```

Comment:
The class Square needs to implement methods calculateArea() and draw(). Currently, it's extending Figure class, but those methods are not defined in the parent class.

Reply:
done